test/YapepBase/Mock/Database/DbConnectionMock.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace YapepBase\Mock\Database;
+
+
+use YapepBase\Database\DbConnection;
+
+/**
+ * Mock class for MysqlConnection
+ *
+ * @codeCoverageIgnore
+ */
+class DbConnectionMock extends DbConnection {
+
+	public function __construct() {
+	}
+
+	protected function connect(array $configuration) {
+		// TODO: Implement connect() method.
+	}
+
+	protected function getBackendType() {
+		// TODO: Implement getBackendType() method.
+	}
+
+	public function getParamType(&$value) {
```

Comment:
Stupid question, but why is this needed? I mean, wouldn't it just work without this method being overridden?

Reply:
Not really. The parent method is protected and I need to call it somehow to test separately.
Actually this method should belong to a another class but I didn't want to refact more now.
It will be fixed in the never coming next version anyway :)